src/main/java/uk/ac/ebi/eva/evaseqcol/controller/admin/AdminController.java

```diff
@@ -76,4 +77,32 @@ public ResponseEntity<?> fetchAndInsertSeqColByAssemblyAccession(
             return new ResponseEntity<>(e.getMessage(), HttpStatus.CONFLICT);
         }
     }
+
+    @Operation(summary = "Add new sequence collection objects",
+            description = "Given FASTA file content, this endpoint will parse the content and use it to construct " +
+                    "seqCol objects with naming convention TEST and eventually save these seqCol objects into the database. " +
+                    "This is an authenticated endpoint, so it requires admin privileges to run it.")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "201", description = "seqCol object(s) successfully inserted"),
+            @ApiResponse(responseCode = "409", description = "seqCol object(s) already exist(s)"),
+            @ApiResponse(responseCode = "404", description = "Assembly not found"),
+            @ApiResponse(responseCode = "400", description = "Bad request. (It can be a bad accession value)"),
+            @ApiResponse(responseCode = "500", description = "Server Error")
+    })
+    @PutMapping(value = "/seqcols/fasta/{insdcAccession}")
+    public ResponseEntity<?> fetchAndInsertSeqColByParsingFastFile(@PathVariable(value = "insdcAccession") String insdcAccession, @RequestBody String fastFileContent) {
```

Comment:
I don't think the string parameter is being used anywhere. It is here to comply with the current object structure.
I guess we should at least change the variant name (and the doc) to make clear it does not have to be an INSDC assembly accession.

Reply:
Positive

Reply:
I agree, it's not being used in the calculations but since we are using a lot of existing code that requires this value, I left it as it is. Was named insdcAccession, since it ultimately sets the insdcAccession value of  AssemblySequenceEntity.
I have updated the parameter name to just accession.